specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4588,11 +4588,21 @@
         "source": {
           "type": "string",
           "description": "The source of the input."
+        },
+        "sourceContext": {
```

Comment:
When were these changes added to the GA REST API? They are breaking changes since skillsets are a round-trippable resource, so I'm hoping they were added before 2019-05-06 was released, and were just forgotten in the Swagger. Can you confirm?

Reply:
These are 'version-neutral' models, which means they are always available... I don't foresee the round tripping issue, because someone with a skillset having a skill not in the sdk, cannot and will not use the SDK, because the skill is not available in it yet.

If that makes sense?

Reply:
I'm not talking about SDK versioning, but about REST API versioning. Since these are version-neutral models, you have to basically *not change them* but instead replace them with "v2" models, or you break backwards compatibility.

The bar for model changes in the REST API is that if you add a new property, it has to be fully optional in such a way that information is not lost if it is omitted or set to null by an older client.

Let's sync up offline to discuss the history of this change before merging this PR.

Reply:
In case you want to check, the internal backend commit is 1cff8c0abea94f27a1aeebef2250adc080c691d4, and was merged on 3/20.  It wasn't in the original skill definition.

Reply:
Discussed offline; We understand the history of this and that it was an intentionally breaking change made before Cognitive Search went GA. We are just updating the Swagger to reflect reality at this point.